In [2]:
import sys 
sys.path.append("..")



In [ ]:
from utils.util import get_today_str
from perplexity import Perplexity
from dotenv import load_dotenv
load_dotenv()

search_client = Perplexity()

date_str = get_today_str()
queries=[
    # 1) 기준금리(한국은행) — 레벨/최근 결정일/최근 12~36개월 추세
    f"{date_str} 개별 site:www.molit.go.kr/USR/policyData/m_34681/lst.jsp?s_category=p_sec_2&srch_usr_titl=Y&psize=100",
]

search_list = []
for i in range(0, len(queries), 5):
    batch = queries[i:i+5]
    res = search_client.search.create(query=batch)
    search_list.append(res)


c:\RAG_COMMANDER\.venv\Lib\site-packages\tqdm\auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html
  from .autonotebook import tqdm as notebook_tqdm


In [4]:
search_list

[SearchCreateResponse(id='fb5e0dd3-0f0f-46ec-bfe7-cc803a83091a', results=[Result(snippet='## 본문\n\n### 마천3재정비촉진구역\n\n### 현황\n\n- 위 치 마천동 283 일대\n\n- 구역면적 134,930.2㎡\n\n- 지역지구 제1·2종일반주거지역\n\n- 사업규모(예정) 지하5층/지상25층, 25개동,연면적 374,650.22㎡, 건폐율 22.85%, 용적률 270.08%\n\n### 추진경위\n\n- 2011.04.21. 재정비촉진계획 변경(마천3구역 정비구역 지정)\n\n- 2013.06.13. 노후도 불충분 사유로 구역 지정 취소\n\n- 2015.01.30. 노후․불량 건축물 현장안전점검\n\n- 2016.11.23. 주민설문조사 결과 보고\n\n- 2017.06.01. 마천3 촉진구역 지정 결정고시\n\n- 2020.06.22. 조합설립인가\n\n- 2022.10.20. 재정비촉진계획 변경 결정고시(서고 제2022-419호)\n\n- 2025.01.23. 재정비촉진계획 변경 결정고시(서고 제2025-54호)\n\n### 향후 추진절차\n\n- 사업시행계획인가', title='마천 3', url='https://www.songpa.go.kr/www/contents.do?key=5833', date='2025-01-23', last_updated='2025-10-12'), Result(snippet='## 본문\n\n### 마천1재정비촉진구역\n\n### 현황\n\n- 위 치 마천동 194-1 일대\n\n- 구역면적 148,498㎡\n\n- 지역지구 제1·2종일반주거지역\n\n- 사업규모 지하2층/지상25층, 44개동 2,413세대, 용적률 227.4%\n\n### 추진경위\n\n- 2011.04.21. 재정비촉진계획 변경(마1 촉진구역 지정)\n\n- 2014.03.13. 노후도 불충분 사유로 구역 지정 취소\n\n- 2019.04.05. 촉진계획(안) 결정요청\n\

[SearchCreateResponse(id='fb5e0dd3-0f0f-46ec-bfe7-cc803a83091a', results=[Result(snippet='## 본문\n\n### 마천3재정비촉진구역\n\n### 현황\n\n- 위 치 마천동 283 일대\n\n- 구역면적 134,930.2㎡\n\n- 지역지구 제1·2종일반주거지역\n\n- 사업규모(예정) 지하5층/지상25층, 25개동,연면적 374,650.22㎡, 건폐율 22.85%, 용적률 270.08%\n\n### 추진경위\n\n- 2011.04.21. 재정비촉진계획 변경(마천3구역 정비구역 지정)\n\n- 2013.06.13. 노후도 불충분 사유로 구역 지정 취소\n\n- 2015.01.30. 노후․불량 건축물 현장안전점검\n\n- 2016.11.23. 주민설문조사 결과 보고\n\n- 2017.06.01. 마천3 촉진구역 지정 결정고시\n\n- 2020.06.22. 조합설립인가\n\n- 2022.10.20. 재정비촉진계획 변경 결정고시(서고 제2022-419호)\n\n- 2025.01.23. 재정비촉진계획 변경 결정고시(서고 제2025-54호)\n\n### 향후 추진절차\n\n- 사업시행계획인가', title='마천 3', url='https://www.songpa.go.kr/www/contents.do?key=5833', date='2025-01-23', last_updated='2025-10-12'), Result(snippet='## 본문\n\n### 마천1재정비촉진구역\n\n### 현황\n\n- 위 치 마천동 194-1 일대\n\n- 구역면적 148,498㎡\n\n- 지역지구 제1·2종일반주거지역\n\n- 사업규모 지하2층/지상25층, 44개동 2,413세대, 용적률 227.4%\n\n### 추진경위\n\n- 2011.04.21. 재정비촉진계획 변경(마1 촉진구역 지정)\n\n- 2014.03.13. 노후도 불충분 사유로 구역 지정 취소\n\n- 2019.04.05. 촉진계획(안) 결정요청\n\n- 2020.01.02. 촉진계획 결정 및 지형도면 고시\n\n- 2022.05.25. 조합설립인가\n\n### 향후 추진절차\n\n- 사업시행계획인가 준비', title='마천 1 - 송파구청', url='https://www.songpa.go.kr/www/contents.do?key=5831', date='2022-05-25', last_updated='2025-10-27'), Result(snippet='### 마천1재정비촉진구역 단독매매 25억6,077만원/3.3㎡\n\n- 재개발\n\n- 조합설립인가\n\n- 대지136㎡\n\n1구역,지분넓고투자가치높음,\n\n- 소유자\n\n- 확인매물 2025. 04. 10.\n\n### 기본 정보\n\n- 매매가\n\n**25억원**\n\n- 관리비0원\n\n- 대지면적136㎡\n\n- 향남향\n\n- 입주가능일2026년 04월 중순 협의 가능\n\n- 매물번호2518996688\n\n- 매물소개전화주시면 상세하게 상담드리겠습니다!!1\n\n- 2025. 4. 10. 최초게재\n\n- 부동산뱅크 제공\n\n### 건축물 정보\n\n로딩중\n\n- 위치서울시 송파구 마천동\n\n- 사업시행단계조합설립인가\n\n- 용적률/건폐율218% / 26%\n\n- 건립총세대수2,018세대\n\n#### 마천역(2024년예정)\n\n- 노선위례선\n\n- 개통2024년 예정\n\n- 거리\n\n- 297m\n\n- 도보 5분\n\n### 주변 분양단지\n\n로딩중\n\n### 중개사\n\n- 중개소\n\n**최성일**송파롯데캐슬공인중개사사무소\n\n- 전화\n\n- 위치서울특별시 송파구 마천로 305 (마천동) 1층\n\n- 등록번호11710-2021-00303\n\n- 중개사 매물최근 3개월 집주인확인 299건\n\n### 중개 보수\n\n로딩중\n\n### 세금\n\n로딩중\n\n### 주변 대중교통\n\n로딩중', title='네이버페이 부동산', url='https://fin.land.naver.com/articles/2518996688', date='2025-04-10', last_updated='2025-04-14'), Result(snippet='서울시 송파구 마천동 2025년 아파트 매매 실거래가\n\n\n\n단지명층수전용면적(㎡)실거래가거래날짜건축년도지번주소\n\n\n\n우방아파트[층수]2[전용면적] 55.32㎡ / 16평[매] 65,000[거래일]2025-7-9[건축]2002[지번]서울시 송파구 마천동 576-0\n\n\n\n우방아파트[층수]2[전용면적] 55.32㎡ / 16평[매] 64,000[거래일]2025-7-8[건축]2002[지번]서울시 송파구 마천동 576-0\n\n\n\n송파파크데일1단지[층수]6[전용면적] 84.85㎡ / 25평[매] 108,500[거래일]2025-7-4[건축]2011[지번]서울시 송파구 마천동 582-0\n\n\n\n송파파크데일2단지[층수]13[전용면적] 84.99㎡ / 25평[매] 110,000[거래일]2025-7-2[건축]2011[지번]서울시 송파구 마천동 607-0... 마천동금호타운1[층수]18[전용면적] 59.93㎡ / 18평[매] 87,000[거래일]2025-6-24[건축]1998[지번]서울시 송파구 마천동 304-0\n\n\n\n아남[층수]1[전용면적] 70.42㎡ / 21평[매] 59,000[거래일]2025-6-10[건축]1992[지번]서울시 송파구 마천동 9-0\n\n\n\n현대그린빌[층수]4[전용면적] 59.67㎡ / 18평[매] 56,500[거래일]2025-6-3[건축]1998[지번]서울시 송파구 마천동 181-0\n\n\n\n송파파크데일1단지[층수]8[전용면적] 59.91㎡ / 18평[매] 90,000[거래일]2025-3-2[건축]2011[지번]서울시 송파구 마천동 582-0', title='서울시 송파구 마천동 2025년 아파트 매매 실거래가 - 아파트봉', url='https://www.aptbong.com/content.php?bt=apt&ct1=110000000&ct2=117100000&ct3=117101140', date='2025-07-09', last_updated='2025-07-22'), Result(snippet='# 송파구 마천동 일대, 신통기획 확정…1천650가구 공급\n\n\n\n기사 스크랩\n\n\n\n공유\n\n\n\n댓글\n\n\n\n클린뷰\n\n\n\n프린트\n\n서울시가 송파구 마천동 183 일대에 최고 39층, 1,650가구 규모의 주거단지가 들어선다.\n\n서울시는 최근 마천동 183 일대에 대한 신속통합기획을 확정했다고 20일 밝혔다.\n\n마천동 183 일대는 거여·마천 재정비촉진지구 중앙에 있음에도 불구하고 오랜 기간 개발이 멈춰 있었다.\n\n또한 마천역, 마천초등학교가 위치해 통근·통학 등 지역 주민의 활동이 많은 곳이지만 복잡한 교통체계로 주민들 불편이 컸다.\n\n다만 대상지 주변으로 거여·마천 재정비촉진지구 개발, 위례선 트램 정거장 신설(2025 예정), 성내천 복원(2028 예정) 등으로 대대적인 변화가 예상된다.... 시는 이러한 지역 특성과 잠재력에 주목해 초등학교와 지하철역을 품은 도시의 편의성, 성내천 복원에 따른 수변(수세권)의 쾌적성을 모두 갖춘 1,650가구 규모의 고품격 주거단지를 조성하는 것을 목표로 신속통합기획을 수립했다.\n\n구체적으로 단지 중앙에 자리잡은 마천초등학교와 마천역, 성내천이 접한 입지 특성을 최대한 활용해 주민의 일상과 여가 공간이 어우러질 수 있도록 영역별 맞춤형 공간을 계획했다. 이를 통해 교육·교통·수변 등 편리하고 쾌적한 생활 환경을 이웃과 함께 누리면서, 단지 주민의 정주환경도 충분히 고려했다.... 이와 함께 마천초와 마천역으로의 접근성을 고려해 기존 보행축이 끊어지지 않도록 동서·남북 보행체계를 연계·강화했다. 학교와 지하철역을 품은 단지 특성을 감안해 단지 내 차량동선 최소화, 주변 도로 확폭 및 일방·양방 혼재된 도로체계 단순화 등 지역 일대 원활한 차량 통행도 고려했다.\n\n또한 위례선 트램 정거장 신설에 따른 유동인구 증가를 고려해 마천로변 휴게공간 조성하고 공공공지·녹지, 어린이집 등 선제적 기반시설 계획도 수립했다. 더불어 단지 내 공공보행통로변 개방형 주민 이용시설, 성내천변 수변특화시설 등 다양한 편의시설도 배치했다.... 이 밖에도 용도지역을 기존 제2종 7층 및 제2종에서 제3종주거지역으로 상향 조정하고 단지 중앙에 최고 39층이 들어설 수 있게 하는 등 유연한 높이 계획을 적용했다. 위례선 개통에 따른 마천역 일대 활성화를 고려해 마천역 연접 준주거지역은 가로 활성화시설 등 복합용도 계획으로 효율적 토지이용을 도모했다.\n\n조남준 서울시 도시공간본부장은 "이번 신속통합기획을 통해 거여?마천 재정비촉진지구의 마지막 퍼즐이 완성되는 만큼 지역에 활력을 부여하는 고품격 주거단지가 되길 기대한다"고 말했다.\n\n양현주기자 hjyang@wowtv.co.kr\n\n서울시는 최근 마천동 183 일대에 대한 신속통합기획을 확정했다고 20일 밝혔다.\n\n마천동 183 일대는 거여·마천 재정비촉진지구 중앙에 있음에도 불구하고 오랜 기간 개발이 멈춰 있었다.... 또한 마천역, 마천초등학교가 위치해 통근·통학 등 지역 주민의 활동이 많은 곳이지만 복잡한 교통체계로 주민들 불편이 컸다.\n\n다만 대상지 주변으로 거여·마천 재정비촉진지구 개발, 위례선 트램 정거장 신설(2025 예정), 성내천 복원(2028 예정) 등으로 대대적인 변화가 예상된다.\n\n시는 이러한 지역 특성과 잠재력에 주목해 초등학교와 지하철역을 품은 도시의 편의성, 성내천 복원에 따른 수변(수세권)의 쾌적성을 모두 갖춘 1,650가구 규모의 고품격 주거단지를 조성하는 것을 목표로 신속통합기획을 수립했다.\n\n구체적으로 단지 중앙에 자리잡은 마천초등학교와 마천역, 성내천이 접한 입지 특성을 최대한 활용해 주민의 일상과 여가 공간이 어우러질 수 있도록 영역별 맞춤형 공간을 계획했다. 이를 통해 교육·교통·수변 등 편리하고 쾌적한 생활 환경을 이웃과 함께 누리면서, 단지 주민의 정주환경도 충분히 고려했다.... 이와 함께 마천초와 마천역으로의 접근성을 고려해 기존 보행축이 끊어지지 않도록 동서·남북 보행체계를 연계·강화했다. 학교와 지하철역을 품은 단지 특성을 감안해 단지 내 차량동선 최소화, 주변 도로 확폭 및 일방·양방 혼재된 도로체계 단순화 등 지역 일대 원활한 차량 통행도 고려했다.\n\n또한 위례선 트램 정거장 신설에 따른 유동인구 증가를 고려해 마천로변 휴게공간 조성하고 공공공지·녹지, 어린이집 등 선제적 기반시설 계획도 수립했다. 더불어 단지 내 공공보행통로변 개방형 주민 이용시설, 성내천변 수변특화시설 등 다양한 편의시설도 배치했다.... 이 밖에도 용도지역을 기존 제2종 7층 및 제2종에서 제3종주거지역으로 상향 조정하고 단지 중앙에 최고 39층이 들어설 수 있게 하는 등 유연한 높이 계획을 적용했다. 위례선 개통에 따른 마천역 일대 활성화를 고려해 마천역 연접 준주거지역은 가로 활성화시설 등 복합용도 계획으로 효율적 토지이용을 도모했다.\n\n조남준 서울시 도시공간본부장은 "이번 신속통합기획을 통해 거여?마천 재정비촉진지구의 마지막 퍼즐이 완성되는 만큼 지역에 활력을 부여하는 고품격 주거단지가 되길 기대한다"고 말했다.\n\n양현주기자 hjyang@wowtv.co.kr', title='송파구 마천동 일대, 신통기획 확정…1천650가구 공급', url='https://www.hankyung.com/article/2024032036925', date='2024-03-20', last_updated='2025-01-06'), Result(snippet='홈\n\n마이페이지\n\n지역필터\n\n로그인\n\n분양정보 친구에게 공유하기\n\n분양정보 공유하기\n\n마천지구(송파파크데일1~2단지) 관심등록\n\n관심등록을 하시면 마천지구(송파파크데일1~2단지)의 모델하우스, 경쟁률, 당첨자명단 등 주요정보를 알림으로 받으실 수 있습니다.\n\n관심등록\n\n상담문의\n\n관심등록\n\n줍줍분양 - 무순위 청약을 알려주는 분양알림 서비스\n\n무단 데이터 및 정보수집 거부 (이메일, 분양정보, 청약정보 등)\n\n제휴문의/고객센터 0504-1360-4582\n\nⓒ All rights are reserved. todaybunyang.com\n\n개인정보처리방침\n\n\n\n회원탈퇴\n\n고객센터 문의전화 0504-1360-4582\n\n대한민국 무순위 청약 정보 알림 서비스\n\n고객센터 업무시간 평일 오전 9시 ~ 오후 6시\n\n개인정보처리방침\n\n\n\n줍줍분양탈퇴\n\n분양은 역시 ⓒ 줍줍분양', title='줍줍분양', url='http://todaybunyang.com/?page=bydetail&s3_bylist_uid=5546', date=None, last_updated='2025-06-11'), Result(snippet="김광섭 이사(JYP홀딩스)가 소개하는\n서울시 송파구 마천동 다세대주택\n\n[투자포인트]\n▶ 5호선 마천역 인접\n▶ 마천5구역 신속통합기획 후보지 선정\n▶ 서울-세종 고속도로 2024년 완공 예정\n\n☎ 02) 2128 – 8504\n\n토마토 집통 채널 정보\n-Genie TV CH188\n-Btv CH158\n-U+tv CH170\n-D'LiVE CH188\n-SK브로드밴드케이블 CH198\n-LG헬로비전 CH163\n-SKY HCN CH414\n\n유튜브를 통해서도 생중계됩니다.\n☞유튜브 : https://www.youtube.com/user/rtomato\n\n#부동산 #송파 #마천... ### Transcript... {ts:0.0} 자 그러면 저희 또 다음 부동산가 보죠 오늘만 답을 마지막 부동산인데요 어떤 부동산일지 화면을 통해 만나 보겠습니다 서울시 송파구 마천동에 위치한 다세대 주택입니다 소개드리는 [음악] 현장에 위치부터 확인해 보겠습니다 가장 가까운역은 지하철 5호선 마천역으로 도보로 이용 가능한 거리의 위치해 있습니다 또한 서울 외곽순환도로에 서하남 ic와 인접해 서울뿐만 아니라 성남 과천 등 수도권 외곽지역으로의 광역교통망을 자랑하는 곳입니다 [음악] 마천동은 [음악] 풍부한 생활 인프라를 자랑하는데요\n{ts:58.739} 본연장의 인근으로 마천중앙시장 주민센터 대형마트 등에 다양한 편의시설이 갖춰져 있어 쾌적한 생활이 가능합니다 또한 거여 마천 뉴타운 마천 5구역 내에 위치해 있어 인근의 위례신도시와 교산신도시의 이미 조성된 다양한 인프라 또한 누릴 수 있습니다 [음악]...  마천동일 때 다양한 개발 오전 소식도 현장의 가치를 높이고 있는데요 마천 5구역이 서울시 신속통합기회 후보지로 선정돼 재개발 사업이 한층 더 빠르게 진행될 것으로 기대됩니다 또한 미래 트램선이 2025년 개통 예정인데요 마천역부터 복정역 남 위례역을 잇는 5.4km 총 12개 정거장을\n지상트램으로 연결해 마천동의 주거 {ts:110.82} 역할을 한층 더 확대시킬 전방입니다 또 하나의 호재 바로 서울 세종고속도로의 개통 소식인데요 2024년이 도로가 개통된다면 경기 남부와 충청권으로 이동이 한층 더 편해질 것으로 판단됩니다 [음악]... 풍부한 생활인프라와 다양한 호재로 미래가치가 더욱더 기대되는 서울시 송파구 마천동에 다세대주택입니다 네 서울시 송파구 마천동 주택 지금부터 보도록 하죠 자 먼저 개요정보부터 부탁드리겠습니다네 본부과는 다세대주택입니다 앞으로도 기대치가 상당히 많은 마천5구역에 위치해 있는 물건이다 이렇게 정리해 드릴 수가 있겠고요 위치는 5호선 마천역에 인접해 있습니다 전용면적을 보시게 되면 59.5평... 정도가 나온다라고 말씀드릴 수가 있겠고요 {ts:163.5} 본 물건 같은 경우는 방 3개 구조입니다 그래서 방 3개의 어떻게 보면 주방과이 거실이 좀 약간 분리되어 있는 어떻게 보면 상당히 좀 용이한 그런 물건이다라고 말씀드릴 수가 있겠고요 본 물건 같은 경우는 2018년도 3월 달에 준공이 되어져 있습니다 매매가는 약간 8억 5천만원이 가능한데 이거 마천5구역에서 예를 들어서 조합원이 되고자 하시는 분들은 여기 같은 경우는 실거주를 하셔야 된다 실버주의 조건이 붙는다 이게 좀 하나에 좀 단점으로 작용할 수 있겠다라고 말씀드릴 수 있겠습니다 자 그러면 건물 특징은 어떻게... 될까요네 본 물건은 2018년도 3월 달에 준공이 된 물건이다 이렇게 말씀드릴 수가 있을 것 같고요 또이 {ts:204.3} 폰 물건 같은 경우는 4천5구역에 대한 어떤 개발 압력이 상당히 심한 지역에 위치해 있는 물건이다라고 말씀드릴 수 있겠습니다이 또 하나는 이쪽 지역이 재개발 지역이기 이후 지정이 되어져 있는 지역이기 때문에 또 여기 같은 경우는 토지거래 허가 구역이거든요 그래서 2년 동안 실거주를 해야지만 조합원의 어떤 자격지위가 자격지위를 갖는다라는 것을 한 번 더 강조를 좀 해드리고요 본부간의 외관을 좀 보시게 되면 상당히 좀 예쁘게 좀 지어져 있는 다세대주택 건물입니다이 또 하나는 안에 이제 내부 시세를 보게 되면... 뭐랄까요 그 뭐 택배함이라든가 뭐 cctv라든가 또 그 엘리베이터 이런 것들이 상당히 좀 예쁘게 좀 구성이 되어져 있는 것도 하나의 좀 {ts:246.599} 특징입니다 또 방 내부 구조가 방 3개의 화장실 2개 또 거실과 주방으로 좀 데워져 있고 또 거실과 좀 약간 분리가 되어져 있어서 어떻게 보면 젊은 부부들한테는 상당히 좋은 그런 어떤 구조를 좀 가지고 있다라고 한번 더 정리를 해드리겠습니다 자 이미 중국 나 있고요 쓰리룸에 화장실 2개가 치고 있는 널찍한 곳으로 살펴 드리고 있습니다 자 그러면 좀 더 밖으로 나가서 주변 자세히 뜯어 볼게요 입지와 호재도 설명해 주시죠네 본 물건은 5호선 마천역에 인접해 있는 물건이다라고 말씀드릴 수가 있겠고요이 어쨌든간에 여기 같은 경우는 송파구... 마천동에 위치해 있는 물가입니다 강남4구 안에 들어가 있는 물건이다 앞으로에 대한 개발 압력이 상당히 좀 유리한 지역에 위치해 있는 물건이다라고 한 번 더 정리를 해 {ts:291.66} 드릴 수가 있겠습니다이 보게 되면 이제 3호선 라인이죠 오금역 라인이 3호선과 5호선이 지나가는데이 마천역과는 약간 세정거장 앞에 위치해 있고요 앞으로 마천역과는 또 위례와 또 연결이 되는 부분이 있기 때문에 마천역 쪽 주변 지역에 상당히 개발 압력이 심하면서도 또 주도할 수 있는 그런 지역에 위치해 있는 물건이다라고 한 번 더 정리해 드리겠습니다 여기는 송파구에 위치해 있기 때문에 인터넷 시설 또한 상당히 좀 우수한데요 바로 옆에 뭐 천마산의 위치해 있고요... 앞쪽으로는 먹자 상권이라든가 주변에 가락 대형마트 또 대형병원에 위치해 있어서 인프레서는 상당히 좀 유리하다 이렇게 좀 보시면 될 거 같고 또 주변 지역을 {ts:332.699} 5구역 내에 위치해 있다라고 말씀드릴 수가 있겠는데 바로 밑이 위례신도시입니다 또 위쪽에 또 교산 신도시에요이 교산 같은 경우는 3기 신도시가 되는 거죠 그 다음에 도로와 교통을이 마천을 계속적으로 거쳐가서 교선에다 만들어 줘야 되기 때문에 어떻게 보면 거의 마천 여기 뉴타운 지역들이 상당히 수혜 지역으로 좀 작용하고 있다는 것도 안에서 특징이다 이렇게 좀 보시면 될 것 같고요 또 이제 마천 5구역 같은 경우는 신속 통합계획 후보지로 좀 선정이 되어져 있습니다... 그러다 보니까 여기 같은 경우는 아직까지는 그 토지거래허가구역으로 묶여 놨다 이렇게 좀 보시면 될 것 같아요 그래서 4천5구역에 대한 기대치가 상당히 높다 이렇게 한 번 더 정리해 드릴 수가 있을 것 같고요 {ts:373.199} 또 하나는 그 유래 트램선입니다 2025년도 이제 그 개통이 되는데 이래 트램선이 개통이 되게 되면 또 마천역과 또 이제 연결이 되는 부분이 있는 거죠 그래서 앞으로 이런 어떤 호적거리가 계속적으로 좀 작용하고 있는 것이 본 지역에 있는 물건들이다 이렇게 좀 보시면 될 것 같고 또 서울세정관 고속도로도 2024년도 이제 완공 예정입니다 이만큼 마천역적 지배인 지역이 왜 계속적으로 마천 마천 하게 되냐면... 이렇게 많은 호적거리들이 작용을 하고 있기 때문에 그만 먹고 기대치가 많아서 앞으로도 그런 기대치에 대한 기대치에 대한 충분한 호응을 할 수 있겠다라고 말씀드릴 수 있겠습니다 그럼 마지막으로 여기 가격전략까지도 살펴볼까요네 {ts:416.16} 본 물건의 예상 매매가는 약한 8억 5천만원 선입니다 쓰리룸 치고이 마천역 쪽 주변지와 마천동 쪽에 8억 5천 정도면 상당히 좀 싼 가격이 나왔다 이렇게 좀 보시면 될 것 같아요 그래서 예상임대가 예상 임대가를 3억으로 좀 정해 놨는데 이거는 어떻게 보면은 실거주를 하시고 추후에 조합원이 될 수 있는 자격을 얻는 것이 상당히 유리한 부분이\n있거든요 그래서 그냥 예선임대가 이런 거 좀 다 지워버리시고 예상 매매가로 아마 내가 뭐 그 조합원이 될 수 있는 자격을 한번 얻어 보겠다 하시는 분들한테는 상당히 좀 좋은 물건이다라고 한 번 더 정리해 드리겠습니다", title='부동산네트워크 : 마천5구역 신속통합기획 후보지 선정! ‘서울시 송파구 마천동 다세대주택’', url='https://www.youtube.com/watch?v=wiXnfYNP8BA', date=None, last_updated='2024-12-20'), Result(snippet="--------------------------------------\n\n김광섭 이사(JYP홀딩스)가 소개하는\n서울시 송파구 마천동 다세대주택\n\n[투자포인트]\n▶ 5호선 마천역 인접\n▶ 서울-세종 고속도로 2024년 완공 예정\n▶ 위례트램선 개통 예정\n\n☎ 02) 2128 – 8504\n\n토마토 집통 채널 정보\n-Genie TV CH188\n-Btv CH158\n-U+tv CH170\n-D'LiVE CH188\n-SK브로드밴드케이블 CH198\n-LG헬로비전 CH163\n-SKY HCN CH414\n\n유튜브를 통해서도 생중계됩니다.\n☞유튜브 : https://www.youtube.com/user/rtomato\n\n#서울 #부동산 #다세대주택... {ts:0} 자 그럼 저희 다음 부동산 가보겠습니다 자 오늘 만나볼 두 번째\n{ts:3} 부동산인데요 어떤 부동산일지 화면을 통해 만나보겠습니다\n{ts:9} 서울시 송파구 마천동에 다세대 주택입니다 [음악]\n{ts:17} 소개드리는 현작의 위치부터 확인해 보겠습니다 가장 가까운 역은 지하철 5호선\n{ts:25} 마천역으로 도보로 이용 가능한 거리에 위치해 있습니다 또한 서울 외곽순환도로에\n{ts:31} 서하남 ic와 인접해 서울뿐만 아니라 성남 과천 등 수도권 및 외곽 지역으로의\n{ts:38} 광역교통망을 자랑하는 곳입니다 [음악] 마천동은\n{ts:54} [음악] 풍부한 생활 인프라를 자랑하는데요 본연장에 인근으로 마천중앙시장\n{ts:61} 주민센터 대형마트 등에 다양한 편의시설이 갖춰져 있어 쾌적한 생활이 가능합니다 또한 거여\n{ts:69} 마천 뉴타운 마천 5구역 내에 위치해 있어 인근의 위례신도시와... {ts:240} 물건입니다 전혀 면적을 보시게 되면 59.5는 약한 11평으로 11평이기 때문에 상당히 좀 양호한 그런\n{ts:247} 물가이다라고 보시면 될 것 같아요 일단 구조가\n{ts:251} 화장실 2개를 좀 구성이 되어져 있고요 있기 때문에 실버주얼로 하기에는 상당히 좀\n{ts:257} 양호하다 이렇게 좀 보시면 될 것 같고요 마천 5구역 내에 위치해 있고 확정이 다 됐습니다 확정이 다 됐기\n{ts:263} 때문에 4천 5구역 내에 위치해 있고 조합은 자격지위를 가질 수 있다\n{ts:268} 단지 뭐냐면 실거주를 해야만 합니다 이렇게 해서 말씀 좀 드리겠습니다\n{ts:273} 준공은 2018년도 3월달에 준공이 된 물건이고요 가격은 약한 6억 5천만원이기 때문에 주변\n{ts:280} 시세 대비 상당히 좀 저렴하게 나온 물건이다라고 정리해 드리겠습니다 자 건물의 특징은 어떻게 될까요네 건물도... {ts:334} 옵션이 상당히 좀 잘 되어져 있는 것도 하나의 특징입니다 각 방마다 또 이제 시스템 에어컨이\n{ts:340} 좀 설치가 되어져 있어서 잘 되어져 있고요 또 이제 재개발로 확정이 되어져 있는 또\n{ts:346} 송파구 마천동이 송파구 강남 3구에 위치해 있는 물건인데 몇몇 가가지고 5초입니다 방 3개짜리인데요\n{ts:353} 근데 요게 또 이제 하나에도 기대치는 재개발이 확정이 되어져 있는 구역 안에 쏙 포함되어 있는 물건이면서\n{ts:359} 대지지분이 약한 11평이나 되는 그런 물건이기 때문에 상당히 좀 양호하고 추위에\n{ts:366} 어떤 조합은 자격을 취득해서 분담금을 계산했을 때도 분담금에서도 상당히 좀 유리한 그런\n{ts:371} 물건 중에 하나다라고 정리해 드리겠습니다 네 2018년에 준공 완료가 된\n{ts:378} 곳이니까 중국 나지는 면역 지나지도 않았죠 깔끔한 건물 거기다가... {ts:382} 널찍한 공간 갖고 있습니다 스리룸에 화장실 2개 뭐 옵션들도 잔뜩 들어가 있다라고 하니까 건물에\n{ts:389} 대한 만족도 충분히 높여 갈 수가 있겠네요 자 그러면 주변 환경이 좀 자세히 보겠습니다\n{ts:394} 입지 여건과 함께 앞으로의 개발에 대한 기대감 더 자세히 알아볼게요 자 입죠 호재도 전해주실까요네 일단\n{ts:401} 송파하고 마천동에 위치해 있는 물건입니다이 강남상고에 위치해 있는 물건이기 때문에 어느 물건보다도\n{ts:407} 상당히 좀 안정적이다 이렇게 좀 말씀을 좀 드릴 수가 있을 것 같고요 여기 같은 경우는 강남 창고 같은\n{ts:413} 경우는 아직 토지거래허가하고 여기에 묶여 있는 지역이다 보니까 일단은 뭐랄까요 실거주로 잡아야 된다라는\n{ts:418} 것이 하나의 좀 단점으로 작용할 수 있다는 것이이 물건이 좀 특징이다라고 말씀드릴 수가 있겠습니다... {ts:424} 본 물권 같은 경우는이 마천역에 인접해 있다라고 말씀 좀 드릴 수가 있을 것 같고요이 마천역 하게 되면\n{ts:429} 또 이제 여기서 이어지는 또 지하철이 또 하나 생기죠 그래서 앞으로도 기대치가 많다라고 이제 한 번 더\n{ts:435} 말씀 좀 볼 수가 있을 것 같고요 주변 여건을 좀 보시게 되면 주변 여건의... {ts:530} 확정이 되어져 있다라고 말씀 좀 드릴 수가 있을 것 같아요 그래서 예전 같은 경우는 뭐 촉진 지역이다\n{ts:536} 촉진지구다라고 해서 어떤 거 재개발 지역에서도 좀 더 쪽지날 수 있도록\n{ts:541} 진행을 했었는데 이거보다 더 빨리 좀 진행을 하겠다라고 하는 것이 바로이 신속통합기획에 대한 확정입니다 그래서\n{ts:547} 안에 이제 그 뭐랄까요 앞서 말씀드린 그 명복동에 위치해 있는 모아타운 보다는 그 범위가 어\n{ts:556} 범위가 훨씬 크는 거죠 범위가 훨씬 크고 약한 5천 세대 이상의 아파트가 이제\n{ts:561} 들어오는 그런 지역이다 이렇게 좀 말씀 좀 드릴 수가 있을 것 같고요이 송사 신속\n{ts:566} 통합 기획의 확진된 곳은 강남참고 중에는 유일하다고 정리를 좀 해 드릴 수가 있을 것 같습니다 또 이제 그\n{ts:573} 유래 트램선이죠이 마천역으로 이어지는이 유래 트램션입니다 이것이 이제 그 복정과 이치로선 복정역과... {ts:680} 지역으로 해서 많은 고속도로가 뚫리고 또 이제 그 지하철도 연결이 되고 트램이라든가\n{ts:686} 신사선도 연결이 되는 그런 부분이 있기 때문에 어떤 그 추후에\n{ts:690} 송파구 마천동 하게 되면 약간 소외되었던 느낌이 있었거든요 근데 이렇게 지하철 또 이렇게 도로에 어떤\n{ts:695} 개통으로 인해서 추위에는 어떤 기대감을 크게 갖고 있는 지역에 위치에 있는 물건이다라고\n{ts:700} 정리해 드리겠습니다 네 마천동요 주변에 인프라엘 일단 잘 되어 있죠 그리고 여기 자체도\n{ts:708} 요새 난리가 난이 신통기획 확정이 되어 있기 때문에요 기대감이 더 높여갈 수 있다는 점을\n{ts:713} 분명히 전해 드렸습니다 자 입죠 호재까지 알아봤고요 마지막으로 가격도 한번 알아볼게요 자 가격 전략은\n{ts:719} 어떻습니까 본 물건은 마천오구요 신속 통합 기획을 하죠 신통기획이라고 해서", title='부동산네트워크 : 마천5구역 신속통합기획 확정! ‘서울시 송파구 마천동 다세대주택’', url='https://www.youtube.com/watch?v=aXxZ_JysPXM', date='2023-06-15', last_updated='2025-09-28'), Result(snippet='서울시 송파구 마천동 183일대 (마천 2구역) 신속통합기획이 확정되었다. 마천동 일원은 2005년 뉴타운 지구로 지정되었으나, 서브프라임과 부동산 침체 등으로 2014년 마천동 183일대 (마천 2구역)는 정비구역에서 해제가 되었다. 마천역과 마천초등학교를 품은 마천 2구역 신속통합기획에 대해 알아보겠다.\n\n**마천동 183일대 (마천 2구역) 신속통합기획**\n\n**1. 교육·교통·자연이 어우러지는 공간 조성**\n\n서울시는 마천초등학교와 마천역, 성내천이 접한 중앙지에 위치한 이 지역을 최대한 활용하여 교육, 교통, 자연이 조화로운 주거환경을 조성하였다. 이를 통해 주민들은 일상 생활과 여가를 함께 즐길 수 있는 맞춤형 공간을 확보하게 되었다.\n\n**2. 보행 연계 및 안전한 교통체계 구축**\n\n마천초등학교와 마천역 주변에는 통근과 통학이 집중되는 지역으로, 이를 고려하여 동서·남북 보행체계를 연계하고 강화하였다. 또한 차량 동선을 최소화하고 주변 도로를 단순화하여 주변 지역의 원활한 차량 통행을 도모하였다.\n\n**3. 미래를 대비한 선제적 기반시설 계획**\n\n위례선 트램 정거장 개통으로 인한 유동인구 증가를 고려하여 마천로변 휴게공간과 공공시설 수요에 유연하게 대응할 수 있는 공공공지와 녹지, 어린이집 등의 선제적 기반시설을 계획하였다.\n\n위례선 트램은 2022년 12월 착공하여 2025년 9월 준공 예정이다.\n\n또한 성내천 복원(’28. 예정) 공사도 진행중이다.\n\n**4. 유연한 도시계획으로 토지이용 및 사업실현성 증대**\n\n지역에 예정된 여러 변화를 고려하여 용도지역을 상향하고 (제2종7층, 제2종→제3종주거), 단지 중앙에는 최고 39층 등 유연한 높이 계획을 적용하였다. 이를 통해 효율적인 토지 이용과 사업 실현성을 높였다.\n\n**5. 조화로운 스카이라인과 도시경관 창출**\n\n지역과 조화로운 스카이라인을 형성하기 위해 단지 내 각 영역별 적정 높이를 적용하였다. 또한 마천역에서 성내천으로 이어지는 공공보행통로와 녹지를 연계하여 통경축을 확보하고 주변으로 열린 도시경관을 창출하였다.\n\n### 마천역 일원 정비구역 현황\n\n## 마천동 183 일대 (마천 2구역) 신속통합기획 참고자료\n\n### □ 대상지 현황\n\n○ 위 치 : 서울시 송파구 마천동 183번지 일대 (74,925.5㎡)\n\n○ 용도지역 : 제2종(7층)일반주거지역, 제2종일반주거지역, 준주거지역\n\n○ 건축물/토지등소유자 : 351동/982명\n\n### □ 신속통합기획 주요 내용\n\n|구분|용도지역|용적률|층수|세대수|\n|--|--|--|--|--|\n|현황|제2종(7층), 제2종, 준주거|약 154%|평균 2.5층(최고 18층)|토지등소유자 982명|\n|신속통합기획(안)|제3종, 준주거|공동주택 300% 이하 복합용지 400% 이하|최고 39층|1,650세대 내외|\n### □ 추진경위 및 향후일정\n\n○ `22.12.30. : 신속통합기획 후보지 선정\n\n○ `23.05.~`24.02. : 신속통합기획 수립 (주민간담회, 주민설명회 등 병행)\n\n○ `24.03. : 신속통합기획안 확정 · 통보 (서울시→자치구)\n\n○ `24.04.~12. : 정비계획(안) 입안절차 추진 및 정비계획 결정\n\n### < 종합구상도 >\n\n### < 건축배치 계획안 >\n\n주변 지역과의 조화 및 단지 내 쾌적한 주거환경을 고려한 단지배치 계획\n\n### < 단지 배치 계획안 >\n\n주변지역 및 학교, 성내천 등과 어우러진 조화로운 단지 형성\n\n', title='마천동 183일대 (마천 2구역) 신속통합기획 추진사항', url='https://review-top7.com/%EB%A7%88%EC%B2%9C%EB%8F%99-183%EC%9D%BC%EB%8C%80-%EB%A7%88%EC%B2%9C-2%EA%B5%AC%EC%97%AD-%EC%8B%A0%EC%86%8D%ED%86%B5%ED%95%A9%EA%B8%B0%ED%9A%8D-%EC%B6%94%EC%A7%84%EC%82%AC%ED%95%AD/', date=None, last_updated='2025-06-03'), Result(snippet="# 거여·마천뉴타운 재개발 속도…4구역 1383가구 짓는다\n\n\n\n기사 스크랩\n\n\n\n공유\n\n\n\n댓글\n\n\n\n클린뷰\n\n\n\n프린트\n\n'새마을'은 공공재개발 후보지\n\n3, 1구역도 조합설립 '순항'\n\n3, 1구역도 조합설립 '순항'\n\n서울 송파구 거여·마천뉴타운 사업이 속도를 내고 있다. 마천4구역 재개발사업이 사업시행인가를 받고, ‘거여새마을’ 구역이 공공재개발 2차 후보지로 선정됐다.\n\n8일 송파구에 따르면 마천4구역 재개발사업이 지난 2일 사업시행인가를 받고 이날 고시됐다.\n\n이 사업을 통해 마천동 323 일원 6만653㎡ 부지에 지하 3층~지상 33층 아파트 1383가구가 조성된다. 조합은 오는 7월부터 시공자 선정 절차에 돌입할 계획이다. 마천동 A공인 관계자는 “거여·마천은 강남3구(강남·서초·송파구) 생활 인프라를 공유하기 쉽다는 장점이 부각되면서 1군 건설사들이 수주에 공들이고 있다”고 말했다.\n\nLH(한국토지주택공사) 등 공공기관이 참여하는 공공재개발을 추진하는 구역도 있다. 거여새마을 구역은 지난달 29일 공공재개발 2차 후보지로 선정됐다. 6만3995㎡ 부지에 아파트 1329가구가 공급된다. 이 구역은 구릉지에 있는 탓에 용적률 상한이 낮고 사업성이 부족해 그간 개발이 원활하게 진행되지 못했다. 주민들은 공공재개발을 통한 용도지역 상향 등으로 사업성이 개선될 것으로 기대하고 있다.\n\n거여동과 마천동 일대에서 진행되는 거여·마천뉴타운은 사업지 규모가 총 104만3000여㎡에 달한다. 거여동 3개 구역(거여2-1, 거여2-2, 거여새마을)과 마천동 5개 구역(마천 1·2·3·4, 마천성당) 등으로 구성됐다.\n\n2005년 뉴타운으로 지정됐지만 이후 부동산 경기 하락 등이 겹치며 잇달아 사업이 좌초하는 등 어려움을 겪었다.\n\n거여·마천뉴타운 중 속도가 가장 빨랐던 ‘e편한세상송파파크센트럴’(거여2-2 재개발·1199가구)은 지난해 6월 입주를 마쳤다. 거여2-1구역에는 ‘송파시그니처롯데캐슬’(1945가구)이 2022년 1월 입주할 예정이다. 이 단지는 2019년 진행된 1순위 청약에서 429가구 모집에 2만3565명이 몰릴 정도로 인기를 끌었다. 실거래가도 상승세를 타고 있다. 송파시그니처롯데캐슬 전용면적 84㎡ 입주권은 지난달 13일 15억8677만원에 신고가 거래가 이뤄졌다. 지난해 1월 12억4000만원에 거래된 뒤 3억4000만원 이상 뛰었다.\n\n다른 구역 정비사업도 진행되고 있다. 마천3구역 재개발 조합은 지난해 6월 조합설립 인가를 받았다. 13만3830㎡를 재개발해 전용 39~99㎡, 2473가구의 대단지를 조성할 계획이다.\n\n마천1구역은 지난해 11월 재개발 추진위원회 승인을 받은 뒤 조합설립을 위한 주민 동의서를 받고 있다. 두 구역 모두 지하철 5호선 마천역을 도보권에 두고 있어 시장의 관심이 높다. 마천동 B공인 관계자는 “오세훈 서울시장 취임으로 각종 재개발·재건축 규제가 풀릴 것이라는 기대가 크다”고 전했다.\n\n장재현 리얼투데이 리서치본부장은 “그동안 거여·마천뉴타운은 송파구 외곽 지역으로 여겨졌지만 최근 인식이 바뀌고 있다”며 “인근 위례신도시 인기가 높아진 데다 e편한세상송파파크센트럴 등이 성공적으로 분양을 마치면서 정비사업 추진이 빨라지고 있다”고 했다.\n\n장현주 기자 blacksea@hankyung.com\n\n8일 송파구에 따르면 마천4구역 재개발사업이 지난 2일 사업시행인가를 받고 이날 고시됐다.\n\n이 사업을 통해 마천동 323 일원 6만653㎡ 부지에 지하 3층~지상 33층 아파트 1383가구가 조성된다. 조합은 오는 7월부터 시공자 선정 절차에 돌입할 계획이다. 마천동 A공인 관계자는 “거여·마천은 강남3구(강남·서초·송파구) 생활 인프라를 공유하기 쉽다는 장점이 부각되면서 1군 건설사들이 수주에 공들이고 있다”고 말했다.\n\nLH(한국토지주택공사) 등 공공기관이 참여하는 공공재개발을 추진하는 구역도 있다. 거여새마을 구역은 지난달 29일 공공재개발 2차 후보지로 선정됐다. 6만3995㎡ 부지에 아파트 1329가구가 공급된다. 이 구역은 구릉지에 있는 탓에 용적률 상한이 낮고 사업성이 부족해 그간 개발이 원활하게 진행되지 못했다. 주민들은 공공재개발을 통한 용도지역 상향 등으로 사업성이 개선될 것으로 기대하고 있다.\n\n거여동과 마천동 일대에서 진행되는 거여·마천뉴타운은 사업지 규모가 총 104만3000여㎡에 달한다. 거여동 3개 구역(거여2-1, 거여2-2, 거여새마을)과 마천동 5개 구역(마천 1·2·3·4, 마천성당) 등으로 구성됐다.\n\n2005년 뉴타운으로 지정됐지만 이후 부동산 경기 하락 등이 겹치며 잇달아 사업이 좌초하는 등 어려움을 겪었다.\n\n거여·마천뉴타운 중 속도가 가장 빨랐던 ‘e편한세상송파파크센트럴’(거여2-2 재개발·1199가구)은 지난해 6월 입주를 마쳤다. 거여2-1구역에는 ‘송파시그니처롯데캐슬’(1945가구)이 2022년 1월 입주할 예정이다. 이 단지는 2019년 진행된 1순위 청약에서 429가구 모집에 2만3565명이 몰릴 정도로 인기를 끌었다. 실거래가도 상승세를 타고 있다. 송파시그니처롯데캐슬 전용면적 84㎡ 입주권은 지난달 13일 15억8677만원에 신고가 거래가 이뤄졌다. 지난해 1월 12억4000만원에 거래된 뒤 3억4000만원 이상 뛰었다.\n\n다른 구역 정비사업도 진행되고 있다. 마천3구역 재개발 조합은 지난해 6월 조합설립 인가를 받았다. 13만3830㎡를 재개발해 전용 39~99㎡, 2473가구의 대단지를 조성할 계획이다.\n\n마천1구역은 지난해 11월 재개발 추진위원회 승인을 받은 뒤 조합설립을 위한 주민 동의서를 받고 있다. 두 구역 모두 지하철 5호선 마천역을 도보권에 두고 있어 시장의 관심이 높다. 마천동 B공인 관계자는 “오세훈 서울시장 취임으로 각종 재개발·재건축 규제가 풀릴 것이라는 기대가 크다”고 전했다.\n\n장재현 리얼투데이 리서치본부장은 “그동안 거여·마천뉴타운은 송파구 외곽 지역으로 여겨졌지만 최근 인식이 바뀌고 있다”며 “인근 위례신도시 인기가 높아진 데다 e편한세상송파파크센트럴 등이 성공적으로 분양을 마치면서 정비사업 추진이 빨라지고 있다”고 했다.\n\n장현주 기자 blacksea@hankyung.com\n\n", title='거여·마천뉴타운 재개발 속도…4구역 1383가구 짓는다', url='https://www.hankyung.com/article/2021040836511', date=None, last_updated='2025-01-20')], server_time=None)]